doc/ref_arch/openstack/chapters/chapter02.rst

```diff
@@ -1776,6 +1777,7 @@ This section is left blank for future use.
      -
      -
      -
+--->
```

Comment:
@pgoyal01 I agree to remove this table, as we have now no issue with numbering. But I think the rendering is not what you expect, the table is still there

Reply:
@karinesevilla Thanks -- deleted table